src/lib/containers/TotalQueryResults.tsx

```diff
@@ -284,9 +285,9 @@ const TotalQueryResults: FunctionComponent<TotalQueryResultsProps> = ({
         ))}
       </div>
       {facetsWithSelection.length > 0 && (
-        <button onClick={clearAll} className="TotalQueryResults__clearall">
+        <Button onClick={clearAll} variant="light" className="TotalQueryResults__clearall">
```

Comment:
https://user-images.githubusercontent.com/17580037/104052770-30d48b00-51b8-11eb-95ae-5c9019ec92c6.mov

Reply:
The above video is incorrect for the 'before' behavior (on the left). It should look the same, but has no change on hover